mixins/http.js

```diff
@@ -60,6 +60,41 @@ const mixin = (superclass) => class extends superclass {
   getPrimaryAttribute() {
     return this.attributes[this.constructor.getPrimaryKey()];
   }
+
+  isNew() {
```

Comment:
I think isNew should not be on http mixins. It's a model property not related to http module